src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java

```diff
@@ -398,7 +398,7 @@ public boolean equals(Object obj)
             && Objects.equals(videoInfo, other.videoInfo)
             && Objects.equals(footer, other.footer)
             && Objects.equals(image, other.image)
-            && Objects.equals(color, other.color)
+            && Objects.equals(color & 0x00FFFFFF, other.color & 0x00FFFFFF)
```

Comment:
```suggestion
            && (color & 0xFFFFFF) == (other.color & 0xFFFFFF)
```

Reply:
the extra zeroes are there for readability

Reply:
it makes it more clear that youre zeroing the first byte

Reply:
I disagree with that, besides that its also inconsistent with how we do this everywhere in JDA. We never do padding with 0's for this and its not needed. That isn't the important part of the suggested change, the part that actually matters is the change from `equals` to `==`.